.github/workflows/build.yml

```diff
@@ -0,0 +1,15 @@
+name: build
+on:
+  pull_request:
+    branches: [master]
```

Comment:
Why only on pull_requests?

I'd expect something like:
```
on:
  push:
    branches: [master]
    tags:
    pull_request:
```

Reply:
was gonna do a separate build + deploy for master in another action once I've got the npm creds etc, but I could potentially make this 
```
on:
  pull_request:
  push:
    branches: master
```

and have a separate deploy only workflow

Reply:
+ tags